docs/release-notes/next/PT-165082649_SC_WS_improve_timeout_msgs

```diff
@@ -0,0 +1,3 @@
+* Sends a more descriptive message to the client if the FSM dies because of a
```

Comment:
This description should be more specific because reading it out of context it is unclear what `client` and `FSM` refer to. Also `dies` is a rather BEAM specific thing.

Reply:
Addressed